src/components/CollapsibleSection/index.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import {View, TouchableOpacity} from 'react-native';
+import Collapsible from './Collapsible';
+import Text from '../Text';
+import {propTypes, defaultProps} from './CollapsibleSectionPropTypes';
+import styles from '../../styles/styles';
+import Icon from '../Icon';
+import {DownArrow, UpArrow} from '../Icon/Expensicons';
+
+class CollapsibleSection extends React.Component {
+    constructor(props) {
+        super(props);
+        this.toggleSection = this.toggleSection.bind(this);
+        this.state = {
+            isExpanded: this.props.isExpanded,
```

Comment:
I didn't see where we are passing this prop and think maybe this should just be `false`?

Reply:
True, we aren't currently using that prop value so I can just remove it and update the initial state to be false here